packages/sdk/src/sdk/api/tracks/TracksApi.ts

```diff
@@ -186,10 +210,29 @@ export class TracksApi extends GeneratedTracksApi {
     this.logger.info('Successfully uploaded track')
     return {
       ...response,
-      trackId: encodeHashId(trackId)
+      trackId: encodeHashId(entityId)!
     }
   }
 
+  /** @hidden
+   * Upload a track
+   */
+  async uploadTrack(
+    params: UploadTrackRequest,
+    advancedOptions?: AdvancedOptions
+  ) {
+    // Validate inputs
+    await parseParams('uploadTrack', createUploadTrackSchema())(params)
```

Comment:
Seems odd to create the schema each time? Shouldn't it be defined once in a types file and referenced?

Reply:
I think probably worth using `refine` or something instead of `max()`